spec/models/unpublishing_spec.rb

```diff
@@ -26,6 +25,11 @@
       expect(subject).to be_invalid
       expect(subject.errors[:type].size).to eq(1)
 
+      # because we haven't given any 'redirects'
```

Comment:
given we have four sets of assertions in this test, perhaps it'd be a good idea to separate them out into separate contexts? that way we could also dispense with the comment.